ws_x_max: 82
ws_y_max: 100
ws_z_max: 3.0

ws_x_min: -37
ws_y_min: -59
ws_z_min: -5.5